.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
This is an example of what Babak was getting at: this change to the gitignore file shouldn't be part of the changes for this particular PR. This is a change that's already in master, so it's just hard to tell what your latest changes are. If we want to follow a proper procedure I want to be able to review just the latest changes otherwise it's too much of a burden. Off hand I have no idea why it appears this way. But you're not merging 22 commits into master, as this suggests. Is it a bug on github? Seems unlikely, probably something wrong with our procedure instead.

Reply:
I see it too. I don't know why this is happening. I was under the impression that I was following the procedure. How do we resolve this?

Reply:
It could be that your branch is out of sync with master. A solution would be for you to merge master into this branch, then do the PR.

Reply:
If you look at "insights" (tab on the top right along with 'code', 'pull requests', ' issues', etc. and then the network graph that shows the graph of branches and merges, it looks correct

Reply:
see https://stackoverflow.com/questions/39048673/github-pull-request-shows-too-many-changes-commits